values.yaml

```diff
@@ -65,6 +65,12 @@ jvb:
       memory: 3000Mi
   # change to /k8s if using matrix.org/docker-jitsi-meet openshift images
   entryPointMountPath: "/entrypoint"
+  # change to the following if using  matrix.org/docker-jitsi-meet openshift images
+  # workDir: /entrypoint
+  # command: /k8s/entrypoint.sh
+  entryPoint:
+    workDir: "/init"
+    command: "/entrypoint/entrypoint.sh"
```

Comment:
Rather than duplicating the directory, make this the script and prefix it with `{{ $.Values.jvb.entryPointMountPath }}` in the STS?